lib/model.js

```diff
@@ -1178,6 +1191,8 @@ Model.prototype.all = function(options) {
  * @param  {Object}                    [options.include[].where] Where clauses to apply to the child models. Note that this converts the eager load to an inner join, unless you explicitly set `required: false`
  * @param  {Array<String>}             [options.include[].attributes] A list of attributes to select from the child model
  * @param  {Boolean}                   [options.include[].required] If true, converts to an inner join, which means that the parent model will only be loaded if it has any matching children. True if `include.where` is set, false otherwise.
+ * @param  {Boolean}                   [options.include[].seperate] If true, runs a seperate query to fetch the associated instances, only supported for hasMany associations
```

Comment:
Ah that's going to be annoying to change and remember if i constantly misspelled it :D